component/src/main/res/drawable/close_white.xml

```diff
@@ -0,0 +1,12 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="18dp"
+    android:height="18dp"
+    android:viewportWidth="18"
+    android:viewportHeight="18">
+  <path
+      android:pathData="M17,17L1,1M17,1L1,17"
+      android:strokeWidth="2"
+      android:fillColor="#00000000"
```

Comment:
Here

The fill color is black

So maybe that's the issue you were getting about the icon being black?